Sources/TuistCore/Graph/GraphTraverser.swift

```diff
@@ -633,10 +633,11 @@ public class GraphTraverser: GraphTraversing {
         }
     }
 
-    func isDependencyPrecompiledDynamicAndLinkable(dependency: GraphDependency) -> Bool {
+    func isDependencyPrecompiledAndLinkable(dependency: GraphDependency) -> Bool {
         switch dependency {
-        case let .xcframework(_, _, _, linking),
-             let .framework(_, _, _, _, linking, _, _),
+        case .xcframework(_, _, _, _):
+            return true
+        case let .framework(_, _, _, _, linking, _, _),
```

Comment:
Are you sure we don't need the same for `.framework`s? Also the the name of the method is misleading because a static framework is precompiled and linkable, but `isDependencyPrecompiledAndLinkable` would return false 🤔